public/javascripts/store.js

```diff
@@ -12,20 +13,20 @@ class Store {
   constructor() {
     this.lineItems = [];
     this.products = {};
-    this.displayOrderSummary();
+    this.displayPaymentSummary();
   }
 
-  // Compute the total for the order based on the line items (SKUs and quantity).
-  getOrderTotal() {
+  // Compute the total for the payment based on the line items (SKUs and quantity).
+  getPaymentTotal() {
     return Object.values(this.lineItems).reduce(
       (total, {product, sku, quantity}) =>
         total + quantity * this.products[product].skus.data[0].price,
       0
     );
   }
 
-  // Expose the line items for the order (in a way that is friendly to the Stripe Orders API).
-  getOrderItems() {
+  // Expose the line items for the payment using products and skus stored in Stripe.
+  getPaymentItems() {
```

Comment:
nit: would getLineItems() make more sense as a method name here since that's what we call the variable in the constructor? "PaymentItems" wasn't very clear to me what that meant.

Reply:
Good catch, yes, will update